libs/core/src/lib/fixed-card-layout/fixed-card-layout.component.ts

```diff
@@ -63,6 +75,92 @@ class CardDropped {
 
 type CardColumn = CardDefinitionDirective[];
 
+@Directive({ selector: '[fdGroup]' })
+export class DndGroup {
+    /** @hidden */
+    _onDndItemFocus$ = new Subject<[number, number]>();
+
+    constructor(public cdr: ChangeDetectorRef) {}
+
+    /** add focus after moving elements with keyboard */
+    focusDndItem(groupIndex: number, itemIndex: number): void {
+        this.cdr.detectChanges();
+        this._onDndItemFocus$.next([groupIndex, itemIndex]);
+    }
+}
+
+@Directive({ selector: '[fdItem]' })
+export class DndItem implements OnInit {
```

Comment:
Let's make more meaningful names - `fdDndItem` or something like that.

Reply:
done